Deprecated/Channel.swift

```diff
@@ -1,39 +0,0 @@
-//
-//  Channel.swift
-//  FBTTUnitTests
-//
-//  Created by Christoph on 1/28/19.
-//  Copyright © 2019 Verse Communications Inc. All rights reserved.
-//
-
-import Foundation
-
-@available(*, deprecated)
-struct Channel: ContentCodable {
```

Comment:
Could you add a comment to the `Hashtag` model that they are also sometimes called `Channels` in the Planetary code? We are still using both names in several places so it might be a nice hint for future developers.

Reply:
👍